processors.go

```diff
@@ -261,8 +261,17 @@ func processFastCGIPass(dirs []Directive) (*caddyhttp.Subroute, []caddyconfig.Wa
 		TransportRaw: caddyconfig.JSONModuleObject(fcgiTransport, "protocol", "fastcgi", nil),
 	}
 
+	/***
+	* If upstream doesn't have scheme explicitly specified as scheme://host, then
+	* assume it's tcp by prefixing it with tcp://. Later we check if the hostname is `unix:`.
+	* If so, then the arguemnt of fastcgi_pass was of the form `unix:/some/path`.
+	**/
```

Comment:
Just curious, why the block style comment here? 🤔

Reply:
Laziness 😂 didn't feel like double tapping the forward slash key